netbox/extras/management/commands/buildschema.py

```diff
@@ -0,0 +1,57 @@
+from json import dumps as json_dumps
+from json import loads as json_loads
+from jinja2 import FileSystemLoader, Environment
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from dcim.choices import (
+    DeviceAirflowChoices, SubdeviceRoleChoices, ConsolePortTypeChoices, PowerPortTypeChoices,
+    PowerOutletTypeChoices, PowerOutletFeedLegChoices, InterfaceTypeChoices, InterfacePoEModeChoices,
+    InterfacePoETypeChoices, PortTypeChoices, WeightUnitChoices
+)
+
+
+class Command(BaseCommand):
+    help = "Generate the NetBox validation schemas."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--console',
+            action='store_true',
+            help="Print the generated schema to stdout"
+        )
```

Comment:
It's not necessary to specify this explicitly: We can output the generated content to STDOUT by default, unless an output filename is specified.